gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/__.java

```diff
@@ -806,6 +805,13 @@ public static <A> GraphTraversal<A, Set<?>> difference(final Object values) {
         return __.<A>start().difference(values);
     }
 
+    /**
+     * @see GraphTraversal#discard()
+     */
+    public static <A> GraphTraversal<A, A> discard() {
```

Comment:
I notice that there is no corresponding `none()` method being replaced here, is this needed?

Reply:
This was originally added to pass a test that looks for steps being implemented in GraphTraversal, but after looking into it again it seems it should just be ignored as none() was previously, so I'll be adding "discard" as an ignore case and removing this function.